example-app/SantokuApp/src/screens/demo/push-notification/usePushNotification.tsx

```diff
@@ -80,15 +112,20 @@ export const usePushNotification = () => {
       return;
     }
     alert('FCM登録トークンを取得してください');
-  }, [token]);
+  }, [channelId, token]);
 
   return {
     authStatus,
     token,
-    requestUserPermission,
+    getPermissionStatus,
+    requestUserPermissionWithoutOptions,
+    requestUserPermissionWithProvisional,
     getToken,
     notifyMessageToAll,
     notifyMessageToMe,
     openSettings,
+    channels,
+    channelKey: channelId,
```

Comment:
I think we can name the whole variable `channelKey`.

Reply:
ea1d814926a917f5bd0bb6c19208ce4c9647f0af